packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -47,9 +59,13 @@ export class ProjectActions {
 
     const browsers = [...(this.ctx.browserList ?? [])]
 
-    await this.api.initializeProject(
-      { ...this.ctx.launchArgs, testingType: this.ctx.wizardData.chosenTestingType }, this.ctx.launchOptions, browsers,
-    )
+    const launchArgs: LaunchArgs = {
+      ...this.ctx.launchArgs,
+      projectRoot: this.ctx.coreData.app.activeProject?.projectRoot!,
```

Comment:
We should avoid avoid `!` if possible, we should probably add to @lmiller1990's lint setup

Instead, if we add `this.ctx.activeProject?.projectRoot` to line 52 above and then use `this.ctx.activeProject.projectRoot` (`ctx.activeProject` is a shorthand for accessing from `coreData.app`) then we shouldn't need the `!` because it's already narrowed